lib/main/bindings/network/network_isolate_binding.dart

```diff
@@ -78,4 +99,19 @@ class NetworkIsolateBindings extends Bindings {
       Get.find<Executor>(),
     ));
   }
+
+  void _bindingSharing() {
+    Get.put(OIDCHttpClient(
+      Get.find<DioClient>(tag: BindingTag.isolateTag)),
+      tag: BindingTag.isolateTag);
+    Get.put(IOSSharingManager(
+      Get.find<KeychainSharingManager>(),
+      Get.find<StateCacheManager>(),
+      Get.find<TokenOidcCacheManager>(),
+      Get.find<AuthenticationInfoCacheManager>(),
+      Get.find<OidcConfigurationCacheManager>(),
```

Comment:
Why no need to bind these with tag?

Reply:
No need for it to be objects from the database, we always reset it every time we instantiate a new isolate.